src/components/my-sales.js

```diff
@@ -62,7 +62,12 @@ class MySales extends Component {
 
   render() {
     const { filter, loading, purchases } = this.state
-    const completedStates = ['withdrawn', 'finalized', 'sellerReviewed']
+    const completedStates = [
+      'withdrawn',
+      'finalized',
+      'sellerReviewed',
+      'ruling'
+    ]
```

Comment:
Since this `completedStated` array is repeated a few times - maybe make it a util?